app/routes/events/components/EventAttendeeStatistics.js

```diff
@@ -0,0 +1,283 @@
+  label: string,
+  distributionData: DistributionDataPoint[],
+}) => {
+  return (
+    <>
+      <h4>{label}</h4>
+      <Flex alignItems="center">
+        <DistributionPieChart
+          dataKey="count"
+          distributionData={distributionData}
+        />
+        <ChartLabel distributionData={distributionData} />
+      </Flex>
+    </>
+  );
+};
+
+const toLocalizedGender = (gender: string) => {
+  if (gender === 'male') return 'Mann';
+  if (gender === 'female') return 'Kvinne';
+  return 'Annet';
+};
+
+const createGenericDataPoint = (
+  selectedDistribution: DistributionDataPoint[],
+  selectedDataPoint: string
+) => {
+  const dataPointEntry = selectedDistribution.find(
+    (entry) => entry.name === selectedDataPoint
+  );
+  if (dataPointEntry) {
+    dataPointEntry.count++;
+  } else {
+    selectedDistribution.push({ name: selectedDataPoint, count: 1 });
+  }
+};
+
+const createGroupDataPoint = (
+  groupDistribution: DistributionDataPoint[],
+  userGroups: number[]
+) => {
+  const isAbakom = userGroups.some((userGroup) =>
+    ABAKOM_GROUP_IDS.includes(userGroup)
+  );
+  const isRevue = userGroups.some((userGroup) =>
+    REVY_GROUP_IDS.includes(userGroup)
+  );
+
+  if (isAbakom) {
+    createGenericDataPoint(groupDistribution, 'Abakom');
+  } else if (isRevue) {
+    createGenericDataPoint(groupDistribution, 'Revy');
+  } else {
+    createGenericDataPoint(groupDistribution, 'Abakus');
+  }
+};
+
+const createRegistrationDateDataPoint = (
+  registrationTimeDistribution: RegistrationDateDataPoint[],
+  registrationDate: Dateish,
+  isRegister: boolean
+) => {
+  const formatedDate = moment(registrationDate).format('DD/MM');
+  const existingTimeEntry = registrationTimeDistribution.find(
+    (entry) => entry.name === formatedDate
+  );
+  if (existingTimeEntry) {
+    if (isRegister) {
+      existingTimeEntry.registrations++;
+    } else {
+      existingTimeEntry.unregistrations++;
+    }
+  } else {
+    registrationTimeDistribution.push({
+      name: formatedDate,
+      registrations: isRegister ? 1 : 0,
+      unregistrations: isRegister ? 0 : 1,
+    });
+  }
+};
+
+const createAttendeeDataPoints = (
+  registrations: Array<EventRegistration>,
+  unregistrations: Array<EventRegistration>
+) => {
+  const attendeeStatistics: {
+    genderDistribution: DistributionDataPoint[],
+    groupDistribution: DistributionDataPoint[],
+    dataTekDistribution: DistributionDataPoint[],
+    komTekDistribution: DistributionDataPoint[],
+    totalDistribution: DistributionDataPoint[],
+    registrationTimeDistribution: RegistrationDateDataPoint[],
+  } = {
+    genderDistribution: [],
+    groupDistribution: [],
+    dataTekDistribution: [],
+    komTekDistribution: [],
+    totalDistribution: [],
+    registrationTimeDistribution: [],
+  };
+
+  const dataTekTotal: DistributionDataPoint = {
+    name: 'Datateknologi',
+    count: 0,
+  };
+  const komTekTotal: DistributionDataPoint = {
+    name: 'Komunikasjonsteknologi',
```

Comment:
As the name of the study has changed I suggest also updating it here, in addition to adding an "m".
```suggestion
    name: 'Kommunikasjonsteknologi og digital sikkerhet',
```

Reply:
✅